asab/metrics/prometheus.py

```diff
@@ -135,12 +29,130 @@ def get_open_metric(self):
 			return text
 
 
+class OpenMetric:
+	def __init__(self):
+		self.Metric = None
```

Comment:
This is weird .... what is this?

Reply:
aha... in order to be able to get all info about Metrics wherever needed (like in warning messages) I gave up passing it through all the functions and wanted to make a class... I should pass the metrics in when I initialize the OpenMetrics instance, I guess...

Reply:
Yes, maybe this is not good design (meaning use of class/object this way).

Reply:
OK, I can return to my cascade of functions